lib/assesments/assessment_2/amna/screens/login/fi_as2_amna_login.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/amna/utils/fi_as2_amna_assets.dart';
+import 'package:flutter_svg/svg.dart';
+import '../../theme/fi_as2_amna_constants.dart';
+part 'widget/_fi_as2_amna_button.dart';
+
+class F1As2AmnaLoginScreen extends StatelessWidget {
+  const F1As2AmnaLoginScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    Widget smallSpace = const SizedBox(
```

Comment:
you can define and initialize this outside build, since build can be called multiple times